server/src/main/java/com/datastrato/gravitino/server/web/rest/TableOperations.java

```diff
@@ -80,7 +81,7 @@ public Response createTable(
       @PathParam("catalog") String catalog,
       @PathParam("schema") String schema,
       TableCreateRequest request) {
-    try {
+    try (PrincipalContext context = Utils.createUserPrincipalContext(httpRequest)) {
```

Comment:
I think you would wrap this for all the operations later on, not just create and alter operations. For example, if you want to support proxy user for hive catalog operations.

Reply:
OK, I will do.

Reply:
Fixed.